docs: added information about the React components (#490)
zendesk/copenhagen_theme: README.md

- Preview requires login so make sure to first run `yarn zcli login -i` if you haven't done that before.

## Assets

> The Copenhagen theme has only some JavaScript assets. You can add other assets to your theme by placing them in the `assets` folder.

Reviewer: ```suggestion
The Copenhagen theme comes with a few JavaScript assets, but you can add other assets to your theme by placing them in the `assets` folder.
```
